src/modules/about-us/BelieveInPower/ImageBackground.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Image from "next/image";
+
+const ImageBackground=()=>
+            <div className="flex items-center">
+            <Image src={"/images/Believe-in-power.png"} 
```

Comment:
need to fix the path here as well. /images/illustrations/Believe-in-power.png. or use SVG for consistency

Reply:
You mean, put the images in `illustrations` folder?